docs/install.rst

```diff
@@ -374,6 +374,19 @@ __ https://gist.github.com/embray/6042780#file-win_register_python-py
 
 .. _builddocs:
 
+Installing Astropy into CASA
+----------------------------
+
+If you want to be able to use Astropy inside `CASA
+<https://casa.nrao.edu/>`_, the easiest way is to run pip from inside CASA.
+Start up ``casapy`` as normal, and type::
+
+    CASA <2>: import pip
+
+    CASA <3>: pip.main(['install', 'astropy', '--user'])
+
+Then close CASA and open it up again, and you should be able to import CASA.
```

Comment:
"you should be able to import CASA." I'm not familiar with CASA but why import CASA after opening CASA? Do you mean import astropy?